src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import moment from 'moment';
+
+const StemDeniedDetails = ({
+  deniedAt,
+  isEnrolledStem,
+  isPursuingTeachingCert,
+}) => {
+  const date = moment(deniedAt).format('MMM D, YYYY');
+
+  return (
+    <>
+      <h1>Your Edith Nourse Rogers STEM Scholarship Application</h1>
+      <div className="vads-u-background-color--primary-alt-lightest vads-u-padding--2 vads-u-margin-bottom--3">
+        <h2 className="claims-alert-header vads-u-font-size--h4">
+          Your application was denied on {date}.
+        </h2>
+      </div>
+      <h2 className="vads-u-font-family--sans vads-u-margin-y--0">
+        Why we denied your claim
+      </h2>
+      <p className="vads-u-margin--0">
+        You must meet all 3 of the eligibility criteria to be considered for the
+        Rogers STEM Scholarship.
+      </p>
+      <h3 className="vads-u-font-family--sans vads-u-margin-bottom--0">
+        You did not meet these criteria for the Rogers STEM Scholarship:
+      </h3>
+      <ul className="stem-ad-list">
+        <li className="stem-ad-list-item">
+          You don’t meet the benefit requirements for the Rogers STEM
+          Scholarship
+          <ul className="stem-ad-list-secondary">
+            <li className="stem-ad-list-item">
+              By law, all Rogers STEM Scholarship recipients must have 6 months
+              or less of Post 9/11 GI Bill (Chapter 33) benefits left (38 U.S.
+              Code § 3320).
+            </li>
+            <li className="stem-ad-list-item">
+              We reviewed your service and school data records currently on
+              file. As of [{date}
```

Comment:
Is this also supposed to be the denied date? It kind of threw me off when the denied date is also used as "a more than 6 months" reference, but you know this form better than I do.